pkg/download/classic/downloader.go

```diff
@@ -74,12 +76,18 @@ func WithAPIs(apis api.APIs) ConstOption {
 	}
 }
 
-func WithAPIContentFilters(apiFilters map[string]contentFilter) ConstOption {
+func WithAPIContentFilters(apiFilters map[string]ContentFilter) ConstOption {
 	return func(d *Downloader) {
 		d.apiContentFilters = apiFilters
 	}
 }
 
+func WithFiltering(b bool) ConstOption {
```

Comment:
just wondering, what does the "Const" in "ConstOption" mean? I don't get it :D

Reply:
from `Construction`
The name is far from perfect. Just propose a better one...

Reply:
`Option`, if that does not work, `DownloaderOption` or no name at all but just the signature

Reply:
Typing is forced to have a compile-time error if the signature of `func NewDownloader(client dtclient.Client, opts ...ConstOption) *Downloader` is changed!

I'll rename it to `Option`